templates/home_page.hbs

```diff
@@ -7,7 +7,7 @@
       <circle cx="4.5" cy="4.5" r="4" fill="none" stroke="currentColor"/>
       <path stroke="currentColor" stroke-linecap="round" d="M11 11L7.5 7.5"/>
     </svg>
-    {{search submit=false instant=settings.instant_search class='search search-full'}}
+    {{search submit=true instant=settings.instant_search class='search search-full'}}
```

Comment:
This is just a question: The search bar design is different than the live theme, think that change was made by Zendesk itself. e.g. The Search logo stays on the left side now.

Reply:
Zendesk has introduced a clear search feature, with the logo on the left side in the upstream master. After rebasing, this is how it looks for us. However, in the upstream version, it is without a submit button:

<img width="878" alt="image" src="https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/b0e75506-1508-467d-9819-72f7395a5892">

Also, we have same experience on mitxonline: https://mitxonline.zendesk.com/hc/en-us